Scripts/Packaging/build_packages.sh

```diff
@@ -23,19 +23,101 @@
 
 set -e
 
+# Default values
 GIT_TOP_LEVEL=$(git rev-parse --show-toplevel)
+ROCM_EXAMPLES_ROOT="$GIT_TOP_LEVEL"
+PACKAGE_NAME="ROCm-SDK-Examples"
+PACKAGE_VERSION="6.2.0"
+PACKAGE_INSTALL_PREFIX="/opt/rocm/examples"
+TEST_PACKAGE_INSTALL_PREFIX="/opt/rocm/examples-test"
+BUILD_DIR="$ROCM_EXAMPLES_ROOT/build"
+DEB_DIR="$BUILD_DIR/deb"
+RPM_DIR="$BUILD_DIR/rpm"
+DEB_PACKAGE_RELEASE="local.9999"
+RPM_PACKAGE_RELEASE="local.9999"
+CPACKGEN=""
```

Comment:
```suggestion
PACKGEN="" # Default is both DEB and RPM
```
Changing name to "PACKGEN", since you're not actually using CPack to build your packages. NOTE: I haven't changed subsequent references.